core/tests/upgrade_account_google_play_tests.rs

```diff
@@ -8,6 +8,7 @@ use test_utils::{
 };
 
 #[test]
+#[ignore]
```

Comment:
close the issue

Reply:
My PR doesn't fully complete the issue unfortunately.